ferrilata-devilline/Controllers/PitchController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Linq;
+using ferrilata_devilline.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ferrilata_devilline.Controllers
+{
+    public class PitchController : Controller
+    {
+        [Route("api/pitches")]
+        public ActionResult Return_Pitches()
+        {
+            var re = Request;
+
```

Comment:
It's so great that you found  the way to do the header extraction magic! If you feel like it, i think that using my authorization-extractor-from-headers extension could make the code here simpler.